providers/cloudns/api.go

```diff
@@ -195,6 +196,7 @@ func (c *cloudnsProvider) get(endpoint string, params requestParams) ([]byte, er
 	// Add auth params
```

Comment:
Please update the TODO

Reply:
I will update TO DO.  because there is other parameter to add in the feature